include/psa/crypto_values.h

```diff
@@ -447,6 +455,12 @@
                         ((type) & PSA_KEY_TYPE_ECC_CURVE_MASK) : \
                         0))
 
+/**@}*/
+
+/** \defgroup iana_groups TLS Supported Groups
+* @{
```

Comment:
`*` needs to be in column 1, otherwise `check-names.sh` gags.

Reply:
Fixed by removing the new defgroup.